### PR TITLE
104 about page

### PR DESCRIPTION
**Pull request by bhryck**

### CODE REVIEW

**Review by ksuhartono97 — APPROVED** [review]

File: ui/src/components/about/VisionMissionSection.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  VStack,
+  Stack,
+  Text,
+  Heading,
+} from "@chakra-ui/layout";
+
+const VisionMissionCard = ({ title, message }) => {
+  return (
+    <Box
+      w={["20em", "30em"]}
```
Comment (ksuhartono97) [inline ui/src/components/about/VisionMissionSection.js:13]: element padding ought to be fixed lol, the width also looks real janky with a bigger container and it looks really spaced far out

**Review by ksuhartono97 — CHANGES_REQUESTED** [review]

File: ui/src/components/about/BeliefsSection.js [hunk 2]
```diff
@@ -0,0 +1,354 @@
...
+          <Box
+            borderWidth="1px"
+            borderRadius="20"
+            shadow="lg"
+            bg="#F8F8F8"
+            px="8"
+            pt="6"
+            pb="6"
+            m="4"
+            h="18em"
```
[the remark this answers is not in the document]
Comment (albertparedandan) [reply ui/src/components/about/BeliefsSection.js:43]: @bhryck

File: ui/src/components/about/StaffSection.js [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+          mb={5}
+        >
+          {title}
+        </Heading>
+      </Center>
+      <VStack>
+        <Box
+          bgColor="#2D4769"
+          w="100%"
+          minH={{ base: "40em", lg: "30em" }}
```
[the remark this answers is not in the document]
Comment (albertparedandan) [reply ui/src/components/about/StaffSection.js:34]: @bhryck

File: ui/src/components/about/BeliefsSection.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (bhryck) [reply ui/src/components/about/BeliefsSection.js]: couldnt reproduce the error but made changes to structure of the grid, should be ok.

[the remark this answers is not in the document]
Comment (ksuhartono97) [reply ui/src/components/about/BeliefsSection.js]: ya looks better now

File: ui/src/components/about/StaffSection.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (ksuhartono97) [reply ui/src/components/about/StaffSection.js:34]: this one also looks ok now

**Review by ksuhartono97 — APPROVED** [review]

**Review by ksuhartono97 — APPROVED** [review]

**Pull request merged**